pkg/controllers/vdb/podfacts.go

```diff
@@ -852,23 +807,6 @@ func parseVerticaNodeName(stdout string) string {
 	return ""
```

Comment:
It's not directly related to your PR, but I saw it when reviewing your change, can you remove `parseVerticaNodeName`? I think we only use it for test. We don't have any non-test operator code that relies on it.